File: ed/l/l.go/examples/whatever/compare.index.go
package main

import (
	"fmt"
	"net/http"
	"time"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		fmt.Printf("\n[%s] New request", time.Now())
		_, busy := r.URL.Query()["busy"]
		if busy {
			time.Sleep(time.Second * 9)
		}
		w.Write([]byte("[go] It works!"))
		fmt.Printf("\n[%s] \t Set response", time.Now())
	})

	http.ListenAndServe(":8080", nil)
}

// curl "http://localhost:8080/?busy"

File: ed/l/l.nodejs/examples/whatever/compare.index.js
const http = require('http');

const handler = (req, res) => {
    console.log('[%s] New request', new Date());
    const t = (req.url === '/?busy') ? 7000 : 0;
    setTimeout(() => res.end('[nodejs] It works!'), t);
};

const app = http.createServer(handler);

app.listen(8000);
// curl 'http://localhost:8000/?busy'
